.NET/Microsoft.Recognizers.Text.DateTime/Dutch/Parsers/DutchHolidayParserConfiguration.cs

```diff
@@ -212,5 +214,9 @@ public override string SanitizeHolidayToken(string holiday)
         private static DateObject Ramadan(int year) => HolidayFunctions.IslamicHoliday(year, HolidayFunctions.IslamicHolidayType.Ramadan);
 
         private static DateObject Sacrifice(int year) => HolidayFunctions.IslamicHoliday(year, HolidayFunctions.IslamicHolidayType.Sacrifice);
+
+        private static DateObject EidAlFitr(int year) => HolidayFunctions.IslamicHoliday(year, HolidayFunctions.IslamicHolidayType.EidAlFitr);

```

Comment:
@tellarin would it make sense to move these to the `BaseHolidayParserConfiguration`, since they will be the same for every language? It seems that quite some (international) holidays could be moved to the Base parser and that just country/language specific holidays could live in the `Locale`HolidayParserConfiguration.

Reply:
@iMicknl, you are right. These should be moved. Do you wanna do it in this PR?

Reply:
I'll open a separate issue for this. Let us know if you'd like to take it.

Reply:
@tellarin don't expect anything from me :-), I won't have a lot of bandwidth for this in the coming weeks. However, if I have some time and it is still open, I will reply to the issue and give it a try.